crates/rerun/src/run.rs

```diff
@@ -305,50 +308,83 @@ async fn run_impl(
     let (shutdown_rx, shutdown_bool) = setup_ctrl_c_handler();
 
     // Where do we get the data from?
-    let rx = if let Some(url_or_path) = args.url_or_path.clone() {
-        match categorize_argument(url_or_path) {
-            ArgumentCategory::RrdHttpUrl(url) => {
-                re_log_encoding::stream_rrd_from_http::stream_rrd_from_http_to_channel(url)
-            }
-            ArgumentCategory::RrdFilePath(path) => {
-                re_log::info!("Loading {path:?}…");
-                load_file_to_channel(&path).with_context(|| format!("{path:?}"))?
+    let rx = if !args.url_or_paths.is_empty() {
+        if 1 < args.url_or_paths.len() {
```

Comment:
That seems pretty awkward?
```suggestion
        if args.url_or_paths.len() > 1 {

Reply:
…however, inverting the `if` and writing this as `if arguments.len() == 1 { … } else { … }` makes even more sense :)